### PR TITLE
Fix typo in controller_manager_interface.stop_controllers()

### PR DESCRIPTION
**Pull request by yli-cpr**
`name` should be `names`

### CODE REVIEW

**Comment by bmagyar** [comment]
Nice catch!
Can I ask you to add a test to `controller_manager_tests` that checks for this please?

**Comment by yli-cpr** [comment]
I actually looked at the unit tests. Currently there is no test that directly tests `controller_manager_interface`. The controller_manager script doesn't use `stop_controllers()`. If we make controller_manager use `stop_controllers()`, we will lose the coverage for `stop_controller()`. So basically we need a new test for the controller_manager_interface.  However, most of the methods in controller_manager_interface have already been covered by the current controller_manager_script test. Adding a full test will just cover 2  more methods: `start_controllers()` and `stop_controllers()` which are one-liners. Not sure it is worth it.

**Comment by bmagyar** [comment]
I think it's not an issue to duplicate tests for a different interface. This is a legit Python interface regardless of whether controller_manager scripts use it directly or not.

BTW extra points for removing the roslib reference in this file :)

**Comment by mathias-luedtke** [comment]
This should get cherry-picked to kinetic and perhaps indigo as well.

**Comment by yli-cpr** [comment]
I don't have melodic environment, so basically I cherry-picked some commits to my current indigo system, and made the unit test work. So please someone please make sure the unit test works in melodic (or it will be covered by a test bot?)

**Comment by bmagyar** [comment]
Travis will do that for us :)

**Comment by yli-cpr** [comment]
```
[Testcase: testcontroller_manager_interface_test] ... ok
[Testcase: testcontroller_manager_scripts_test] ... ok
[ROSTEST]-----------------------------------------------------------------------
[controller_manager_tests.rosunit-controller_manager_interface_test/test_scripts][passed]
[controller_manager_tests.rosunit-controller_manager_scripts_test/test_scripts][passed]
SUMMARY
 * RESULT: SUCCESS
 * TESTS: 2
 * ERRORS: 0
 * FAILURES: 0
```

**Review by mathias-luedtke — APPROVED** [review]
No bonus points though ;)

**Comment by yli-cpr** [comment]
Thanks!

**Comment by yli-cpr** [comment]
I don't have write access to merge it. @bmagyar , please help if you think it is OK to merge

**Review by bmagyar — APPROVED** [review]

File: controller_manager/src/controller_manager/controller_manager_interface.py [hunk 1]
```diff
@@ -36,15 +36,12 @@ def reload_libraries(force_kill, restore = False):
...
-        to_start = []
-        for c, s in zip(originally.controllers, originally.state):
-            if s == 'running':
-                to_start.append(c)
-        switch_srv(start_controllers = to_start,
-                   stop_controllers = [],
-                   strictness = SwitchControllerRequest.BEST_EFFORT)
+        for c in originally.controller:
+            load_srv(c.name)
+        to_start = [c.name for c in originally.controller if c.state == 'running']
```
Comment (bmagyar) [inline controller_manager/src/controller_manager/controller_manager_interface.py:41]: Sweet!

**Review by bmagyar — APPROVED** [review]

**Comment by bmagyar** [comment]
Thanks for sticking with us, pleasure to do business with you! :+1:

**Pull request merged**